tests/PySys/software-management-end-to-end/sm-apt-install-fail/run.py

```diff
@@ -0,0 +1,33 @@
+from pysys.basetest import BaseTest
+
+import time
+
+"""
+Validate end to end behaviour for failing installation
+
+When we install a package that cannot be installed with the apt package manager
+Then we receive a failure from the apt plugin
+"""
+
+import time
+import sys
+
+sys.path.append("software-management-end-to-end")
+from environment_sm_management import SmManagement
+
+
+class PySysTest(SmManagement):
+    def setup(self):
+        super().setup()
+
+    def execute(self):
+
+        self.trigger_action("does_not_exist", "5446165", "::apt", "notanurl", "install")
```

Comment:
We should find a way to avoid passing that `5446165`. We can do a lookup into the Software Repository with the following query: `{{url}}/inventory/managedObjects?type=c8y_Software` and then can find the id for the software that we're looking for.

Reply:
Good idea, as discussed we resolve that in another PR.